x/treasury/handler.go

```diff
@@ -34,12 +34,18 @@ func handleMsgSend(ctx sdk.Context, k keeper.Keeper, msg types.MsgSend) sdk.Resu
 		return err.Result()
 	}
 
-	ctx.EventManager().EmitEvent(
+	ctx.EventManager().EmitEvents(sdk.Events{
+		sdk.NewEvent(
+			types.EventTypeMsgSend,
```

Comment:
```suggestion
			types.EventTypeSend,
```

Reply:
ok. updated the code types.EventTypeSend